metrictank.go

```diff
@@ -398,6 +398,8 @@ func main() {
 		handlers = append(handlers, notifierNsq.New(*instance, metrics))
 	}
 
+	// The notifierKafka handler will block here until it has processed the backlog of metricPersist messages.
+	// it will block for 60seconds at most.
```

Comment:
`.start()` seems to get called in `notifierKafka.New()`, so actually it's `New()` which blocks and not the next line.